### ISSUE TITLE
UMA Dynamic Character Builds Skeleton outside of prefab in scene

### ISSUE THREAD

**Issue by Farhannibal** [umasteeringgroup/UMA#40]
Attempted to create a UMA race using a low poly model as a base mesh. I followed this tutorial: https://www.youtube.com/watch?v=ox6iZUfuIso that explains how to create a custom UMA race using a base model.

[Video of Problem](https://zippy.gfycat.com/TerrificFrayedCowrie.webm)

When I attempt to change the Dynamic character using the simple assets found in the "Getting Started" folder, I can successfully see it in the list of races, but when I switch it out, it seems to put the bones outside of the character.

Should note that the FBX file imported into Unity generated the skeleton, and named the root bone "Back" instead of "Global" like the example mesh. Otherwise unsure if any problems in the race files, followed the tutorial previously linked for that process.

Using UMA 2.6 and Unity 2017.0f3.

I get these specific errors and warnings when I switch to the custom race:
![Image](http://i.imgur.com/kKtuPXq.png)
![Image](http://i.imgur.com/dGBJXVu.png)

This is the function where the warning occurs:
![Image](http://i.imgur.com/WOmNmFs.png)

**Comment by kenamis** [comment]
Are you still having this issue? I thought remember someone was helping you with it?

**Comment by Farhannibal** [comment]
Yes, this issue still persists when trying to port in that model as a race. I don't believe anyone commented on this issue as of yet.

**Comment by kenamis** [comment]
Would you be okay with sending me the mesh file for me to take a look? if
so, you can email it to me at kenan.chabuk@gmail.com

**Comment by Farhannibal** [comment]
All right, I sent an email with the model, thanks a lot for looking into this.

**Comment by kenamis** [comment]
It looks like you missed some steps from Secret Anorak's tutorial.  Specifically, https://youtu.be/6bB4DIGiW4Q?t=171 .  Try that and see if that helps.  I did that and the slot was able to generate but you'll have to redo the avatar/tpose.  Also, I noticed varying scales set on your mesh and bones.  You'll want those all at 100% for export.  So if you're using scale to adjust the mesh, then bake the vertex position and reset the scales back to 100%.
